Flow docs enhancement: add examples for Grafana Cloud (#3288)
grafana/agent: docs/sources/flow/reference/components/prometheus.remote_write.md

* `prometheus_remote_storage_exemplars_in_total` (counter): Exemplars read into
  remote storage.

## Examples

> This example creates a `prometheus.remote_write` component that sends received
> metrics to a local Mimir instance:

Reviewer: ```suggestion
The following examples show you how to create `prometheus.remote_write` components that send metrics to different destinations.

### Send metrics to a local Mimir instance

You can create a `prometheus.remote_write` component that sends your metrics to a local Mimir instance:
```

  forward_to = [prometheus.remote_write.staging.receiver]
}
```

> 
> 

Reviewer: ```suggestion

```

Remove the extra blank line

> 
> This example creates a `prometheus.remote_write` component that sends received
> metrics to Grafana Cloud with username and password injected through environment
> variables:

Reviewer: ```suggestion
### Send metrics to a hosted endpoint

You can create a `prometheus.remote_write` component that sends your metrics to a hosted endpoint, for example Grafana Cloud. The username and password are injected through environment variables.
```

  forward_to = [prometheus.remote_write.staging.receiver]
}
```

> ### Send metrics to a hosted endpoint

Author: ```suggestion
### Send metrics to Grafana Cloud
```

> You can create a `prometheus.remote_write` component that sends your metrics to a hosted endpoint, for example Grafana Cloud. The username and password are injected through environment variables.

Author: ```suggestion
You can create a `prometheus.remote_write` component that sends your metrics to Grafana Cloud. The Prometheus username and the Grafana Cloud API Key are injected in this example through environment variables.
```

```river
prometheus.remote_write "default" {
  endpoint {
    url = "https://prometheus-xxx.grafana.net/api/prom/push"
      basic_auth {
        username = env("PROMETHEUS_USERNAME")

>         password = env("PROMETHEUS_PASSWORD")

Author: ```suggestion
        password = env("GRAFANA_CLOUD_API_KEY")
```
